training/build.gradle

```diff
@@ -31,7 +31,16 @@ bintray {
 }
 
 dependencies {
+  compile 'org.scala-lang:scala-library:2.10.4'
   compile project(':core')
+
+  compile ('org.apache.avro:avro:1.7.7') {
+    transitive = false
```

Comment:
Turns out we have some transitive dependencies conflicts previously (jackson lib probably). It's fine to just exclude transitive here as Avro's dependencies are minimal and they should also be provided by either spark or hadoop.